pkg/virt-controller/services/template_test.go

```diff
@@ -421,6 +422,7 @@ var _ = Describe("Template", func() {
 			},
 				Entry("on amd64", "amd64", "/usr/share/OVMF"),
 				Entry("on arm64", "arm64", "/usr/share/AAVMF"),
+				Entry("on ppc64le", "ppc64le", "/usr/share/OVMF"),
```

Comment:
Do we also need this on line 1140?
https://github.com/kubevirt/kubevirt/blob/c7d0d0cf8a6405bbb69710844fa0bb4d1d1bfc4e/pkg/virt-controller/services/template_test.go#L1138-L1141

Reply:
@rthallisey  It was identified that support for ppc64le is no longer being supported but since I had already added it during the initial dev, @rmohr was thinking we could just leave it in for now